test/jit/test_models.py

```diff
@@ -395,11 +396,11 @@ class Config:
     def test_snli(self):
         self._test_snli(self, device='cpu')
 
-    if 'fbgemm' in torch.backends.quantized.supported_engines:
-        # Suppression: this exercises a deprecated API
-        @suppress_warnings
-        def test_snli_quantized(self):
-            self._test_snli(self, device='cpu', quantized=True)
+    @skipIfNoFBGEMM
+    # Suppression: this exercises a deprecated API
```

Comment:
Do annotations function correctly if there's a comment in between them? o.O

Reply:
Good point, let me check

Reply:
They do, run the following to check:
```
#!/usr/bin/env python
from functools import wraps
def dummy_decorator(f):
    @wraps(f)
    def wrapper(*args, **kwargs):
        print("wrapper was called")
        return f(*args, **kwargs)
    return wrapper

@dummy_decorator
# This is random comment
def foo():
    print("Foo")


if __name__ == '__main__':
    print("Hello World")
    foo()
```